src/container.c

```diff
@@ -106,12 +107,22 @@ static int container_setup_volume(struct hyper_container *container)
 		const char *filevolume = NULL;
 		vol = &container->vols[i];
 
-		if (vol->scsiaddr)
-			hyper_find_sd(vol->scsiaddr, &vol->device);
+		sprintf(mountpoint, "./%s", vol->mountpoint);
+		if (vol->scsiaddr) {
+			if (!strcmp(vol->scsiaddr, INIT_VOLUME_MOUNTPOINT_SCSIADDR)) {
+				fprintf(stdout, "init mountpoint %s \n", mountpoint);
```

Comment:
How about changing to more clearer log, can not get enough information from this log without check codes.